WooCommerce/WooCommerceTests/POS/ViewModels/ItemListViewModelTests.swift

```diff
@@ -49,54 +49,82 @@ final class ItemListViewModelTests: XCTestCase {
 
     func test_itemListViewModel_when_populatePointOfSaleItems_then_state_is_loaded() async {
         // Given
+        let expectedItem = Self.makeItem()
+
+        XCTAssertEqual(sut.state, .loading)
+
+        // When
+        await sut.populatePointOfSaleItems()
+
+        // Then
+        XCTAssertEqual(sut.state, .loaded([expectedItem]))
+    }
+
+    func test_itemListViewModel_when_populatePointOfSaleItems_has_no_items_then_state_is_loaded_empty() async {
+        // Given
+        let itemProvider = MockPOSItemProvider()
+        itemProvider.shouldReturnZeroItems = true
+        let sut = ItemListViewModel(itemProvider: itemProvider)
+        let expectedResult = ItemListViewModel.EmptyModel(title: "No products",
```

Comment:
If as you said you move these static model descriptions to `EmptyModel`, you could use it here as well instead of retyping all these strings.